cpp/unit/type_pair.cpp

```diff
@@ -0,0 +1,16 @@
+// Copyright 2015-2015 the openage authors. See copying.md for legal info.
+
+#include "type_pair.h"
+
+namespace openage {
+
+UnitType::UnitType() {}
+
+bool UnitType::match(Unit *) {
```

Comment:
btw, if you got unused arguments, you can write `match(Unit */*lolname*/)`, up2you.